src/Symfony/Bundle/FrameworkBundle/Tests/Functional/app/Uid/config_disabled.yml

```diff
@@ -5,3 +5,8 @@ framework:
     http_method_override: false
     uid:
         enabled: false
+
+services:
+    Symfony\Bundle\FrameworkBundle\Test\ExceptionListener:
+        tags:
+            - { name: kernel.event_listener, event: kernel.exception }
```

Comment:
Would it make sense instead to let the `ExceptionListener` class implement the `EventSubscriberInterface`? That would make the registration in application easier.

Reply:
It is just a class for testing. I think it should be opt-in.

Reply:
It's still opt-in as it won't do anything if I don't register it as a service in my application, right? But assuming a standard Symfony application I would only have this configuration code instead:

```yaml
services:
    Symfony\Bundle\FrameworkBundle\Test\ExceptionListener: ~
```

Reply:
Thank you. 

I've updated the code

Reply:
I had to make it: 

```yaml
services:
    Symfony\Bundle\FrameworkBundle\Test\ExceptionSubscriber:
        tags:
            - {name: kernel.event_subscriber}

```

Because EventSubscriberInterface was not automatically added the tag in this test env.